iis-wf/iis-wf-referenceextraction/src/test/resources/eu/dnetlib/iis/wf/referenceextraction/project/data/document_to_project.json

```diff
@@ -386,7 +386,7 @@
   "documentId": "50|core_ac_uk__::2fea53c390909c2640b2d1a94d53c0a7",
   "projectId": "40|inca________::1e5e62235d094afd01cd56e65112fc63",
   "confidenceLevel": 0.8,
-  "textsnippet": "This work was supported by INCA (plbio 2010-216 and INCa-DGOS-Inserm 6046)"
+  "textsnippet": "This work was supported by INCA (plbio 2010-216 and INCa-DGOS-Inserm 6046) "
```

Comment:
This is quite strange, and noticed this already before: sometimes when I change the set of funders covered with integration tests I deal with the cases as the one highlighted here. White character (space in this case) either shows up or is removed from the end of the `textsnippet` field. It does not happen when the set of input projects/documents remains unchanged, only when I alter it.

@johnfouf do you have any clue what could be the reason for that? It is not super crucial, it is related to `textsnippet` auxiliary field, but it makes every iteration of changing the integration tests set more cumbersome because test fails, I need to find the reason of the failure and quite often it comes down to either removing or adding space character at the end of the `textsnippet` line. Could we e.g. truncate this field content to get rid of this white character related indeterminacy?